test/e2e/scheduling_test.go

```diff
@@ -252,6 +252,137 @@ var _ = ginkgo.Describe("propagation with label and group constraints testing",
 			framework.RemoveClusterPropagationPolicy(karmadaClient, crdPolicy.Name)
 		})
 	})
+	ginkgo.Context("Job propagation testing", func() {
+		var groupMatchedClusters []*clusterv1alpha1.Cluster
+		var targetClusterNames []string
```

Comment:
Hi, @XiShanYongYe-Chang 
Do we need this test?

Reply:
Just wait for CI pass, we can merge it.

Reply:
Sorry, I misunderstood. 

This one can stay.

Reply:
All right, thank you!